src/components/TextField/TextField.tsx

```diff
@@ -252,6 +256,17 @@ class TextField extends React.PureComponent<CombinedProps, State> {
       </div>
     ) : null;
 
+    const clearButtonMarkup =
```

Comment:
I feel like it might make sense to not show the button when the value is empty?

Reply:
This is a good question for design. @sarahill, do you think it makes sense for the clear button to always be present, or for it to appear once the merchant has typed into the field?

Reply:
I agree with @kvendrik. The clear button only appears once the merchant has typed.